docs/release-notes/8.14.asciidoc

```diff
@@ -0,0 +1,62 @@
+* Adds alert suppression for New Terms rules ({kibana-pull}178294[#178294]).
+* Adds alert suppression for EQL rules with non-sequence queries ({kibana-pull}176422[#176422]).
+* Allows you to edit value lists from the UI, anywhere you use them ({kibana-pull}179339[#179339]).
+* Adds a **Setup guide** markdown field to custom rules ({kibana-pull}178131[#178131]).
+
+[discrete]
+[[enhancements-8.14.0]]
+==== Enhancements
+* Removes the "Technical preview" tag for {esql}; it is now generally available ({kibana-pull}180838[#180838]).
+* Allows you to add calculated values to an {esql} rule's highlighted fields ({kibana-pull}177746[#177746]).
+* Connects {esql} functionality in {elastic-sec} to the `general:enableESQL` advanced setting ({kibana-pull}181616[#181616]).
+* Removes the "Technical preview" tag for custom query rule alert suppression; it is now generally available ({kibana-pull}181279[#181279]).
+* Makes conversations with Elastic AI Assistant persist across sessions ({kibana-pull}173487[#173487]).
+* Adds conversation streaming for Elastic AI Assistant ({kibana-pull}180095[#180095]).
+* Adds support for Anthropic Claude 3 to the Amazon Bedrock connector, and makes it the default model ({kibana-pull}179304[#179304]).
+* Adds an **AI Assistant** settings section to the **Management** menu ({kibana-pull}176656[#176656]).
+* Updates the **AI Assistant** design from modal to flyout ({kibana-pull}176657[#176657]).
+* Adds the `_source` field to the alert details flyout's JSON view ({kibana-pull}180477[#180477]).
+* Improves the UI for row renderers in Timeline ({kibana-pull}180669[#180669]).
+* Allows data collected by Auditbeat to appear in Session View ({kibana-pull}179985[#179985]).
+* Improves the visual appearance of the asset criticality alert column ({kibana-pull}180868[#180868]).
+* Adds an advanced setting that allows you to turn off alert enrichment from memory scanning for malicious behavior alerts ({kibana-pull}180636[#180636]).
+* Adds an advanced setting that lets you turn off a performance optimization that makes malware on-write and file event processing asynchronous ({kibana-pull}179179[#179179]).
+* Makes some of the flyout's state persist for alert and event details ({kibana-pull}178746[#178746], {kibana-pull}179511[#179511]).
+* Limits the alerts that can affect an entity's risk score to the 10,000 riskiest ({kibana-pull}178324[#178324]).
+* Adds a tooltip to the **Asset Criticality** section of the entity details flyout ({kibana-pull}176927[#176927]).
+* Updates MITRE ATT&CK framework to version 14.1 ({kibana-pull}174120[#174120]).
+* Allows you can choose whether {elastic-defend} scans files when they're modified or executed ({kibana-pull}179176[#179176]).
+* Allows you to configure {elastic-defend}'s malware protection to automatically register {elastic-defend} as the antivirus software for Windows endpoints ({kibana-pull}180484[#180484]).
```

Comment:
> Allows you to configure {elastic-defend}'s malware protection to automatically register {elastic-defend} as the antivirus software for windows endpoints

Slight adjustment, maybe "sync" rather than "configure" (Allows you to sync {elastic-defend}'s malware protection to automatically register {elastic-defend} as the antivirus software for windows endpoints)? Or, "Allows you to automatically register {elastic-defend} as the antivirus software for windows endpoints when {elastic-defend}'s malware protection is set to prevent."

Reply:
I like the second one! Thanks for the review 👍🏼 

```suggestion
* Allows you can choose whether {elastic-defend} scans files when they're modified or executed ({kibana-pull}179176[#179176]).
* Allows you to automatically register {elastic-defend} as the antivirus software for Windows endpoints when {elastic-defend}'s malware protection is set to prevent ({kibana-pull}180484[#180484]).
```